### ISSUE TITLE
Feature: new rule for avoid variables with same description

### ISSUE THREAD

**Issue by evairmarinho** [terraform-linters/tflint#1180]
In Terraform, is common copy and paste variables, only changing the type and the alias.

### Example:

```hcl
variable "db_username" {
type = string
description = "Username of database"
}
```
Crating a new variable for db password:

```hcl
variable "db_username" {
type = string
description = "Username of database"
}

variable "db_password" {
type = string
description = "Username of database"
}
```

 ### New rule for tflint
```sh
tflint --only=terraform_documented_variables_with_different_description

[Warning] Same description of terraform variables in variables.tf
```

**Comment by bendrucker** [comment]
This would fit fine as a feature of the existing variable rule. Some of the the rules could really use a rename. Like the rule name should be `terraform_variable_description` and the attribute here would be `unique = true`. That way the rule could also have `required = true` (by default). Users could choose to enforce unique descriptions but not require all variables have one. 

I'll rename and deprecate a bunch of rules in one batch. Happy to review a PR adding the `unique` attribute to the existing rule with its existing name.
